doc/source/contributing.rst

```diff
@@ -286,13 +286,21 @@ Some other important things to know about the docs:
   overviews per topic together with some other information (what's new,
   installation, etc).
 
-- The docstrings follow the **Numpy Docstring Standard** which is used widely
+- The docstrings follow the **Numpy Docstring Standard**, which is used widely
   in the Scientific Python community. This standard specifies the format of
   the different sections of the docstring. See `this document
   <https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt>`_
   for a detailed explanation, or look at some of the existing functions to
   extend it in a similar manner.
 
+- The ``.rst`` files are used to automatically generate Markdown and HTML versions
+  of the docs. For this reason, please do not edit ``CONTRIBUTING.md`` directly, but 
+  instead make any changes to ``doc/source/contributing.rst``. Then, to generate
+  ``CONTRIBUTING.md``, use [pandoc](http://johnmacfarlane.net/pandoc/) with the 
+  following command:
+
```

Comment:
`conda install pandoc` throws an error on my machine because the package isn't found in any current osx-64 channels. Can you specify a channel in `ci/requirements_dev.txt` and `ci/requirements_all.txt`? Or can we get around this problem by adding pandoc to the pandas channel?